### PR TITLE
[SYCL][ESIMD] Disable tests failing in pre-commit CI

### PR DESCRIPTION
**Pull request by aelovikov-intel**

### CODE REVIEW

File: SYCL/ESIMD/api/replicate_smoke.cpp [hunk 1]
```diff
@@ -12,6 +12,9 @@
 //
 // The test checks main functionality of the esimd::replicate_vs_w_hs function.
 
+// Temporarily disable until the failure is being investigated.
```
Comment (aelovikov-intel) [inline SYCL/ESIMD/api/replicate_smoke.cpp]:
```suggestion
// Temporarily disable while the failure is being investigated.
```

File: SYCL/ESIMD/api/simd_view_select_2d_int.cpp [hunk 2]
```diff
@@ -13,6 +13,9 @@
 // Smoke test for 2D region select API which can be used to represent 2D tiles.
 // Tests int types.
 
+// Temporarily disable until the failure is being investigated.
```
Comment (aelovikov-intel) [inline SYCL/ESIMD/api/simd_view_select_2d_int.cpp]:
```suggestion
// Temporarily disable while the failure is being investigated.
```

File: SYCL/ESIMD/ext_math.cpp [hunk 3]
```diff
@@ -15,6 +15,9 @@
 // - math function - sin, cos, ..., div_ieee, pow
 // - SYCL vs ESIMD APIs
 
+// Temporarily disable until the failure is being investigated.
```
Comment (aelovikov-intel) [inline SYCL/ESIMD/ext_math.cpp]:
```suggestion
// Temporarily disable while the failure is being investigated.
```

File: SYCL/ESIMD/matrix_transpose_glb.cpp [hunk 4]
```diff
@@ -10,6 +10,9 @@
 // RUN: %clangxx -fsycl %s -o %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 
+// Temporarily disable until the failure is being investigated.
```
Comment (aelovikov-intel) [inline SYCL/ESIMD/matrix_transpose_glb.cpp]:
```suggestion
// Temporarily disable while the failure is being investigated.
```

**Review by v-klochkov — APPROVED** [review]

**Comment by aelovikov-intel** [comment]
@intel/llvm-gatekeepers this is ready to merge - the failures are on tests not touched by this PR and the ones that are modified here stopped failing.

**Pull request merged**